lib/connection.js

```diff
@@ -1392,6 +1392,25 @@ Connection.prototype.syncIndexes = async function syncIndexes() {
  * Switches to a different database using the same connection pool.
  *
  * Returns a new connection object, with the new db.
+ * 
+ * ####Example
+ *    function createMultiTenatConnection(tenantId, modelName, schema) {
+  *     mongoose.connect('mongodb://localhost:27017/test');
```

Comment:
Shouldn't put `mongoose.connect()` here, you don't want to call `mongoose.connect()` every time you create a multi tenant connection